UnrealPlugin/ThirdPersonMPCloudDeploy.md

```diff
@@ -91,56 +92,12 @@ This option runs your game server(s) in docker containers, isolating them from e
 * Select __Add Build__
 * Skip ahead to the __Configure Regions__ section below
 
-### Setting up a Linux Dedicated Server on PlayFab
-
-A more detailed Linux guide can be found [here](https://github.com/PlayFab/MpsAgent/blob/main/LocalMultiplayerAgent/MultiplayerSettingsLinuxContainersOnWindowsSample.json)
-
-During testing the following Dockerfile + startup.sh script worked excellent with the PlayFab Linux VM:
-
-#### The Linux Dockerfile:
-
-```Dockerfile
-FROM ubuntu:18.04
-
-# Unreal refuses to run as root user, so we must create a user to run as
-# Docker uses root by default
-RUN useradd --system ue
-USER ue
-
-EXPOSE 7777/udp
-
-WORKDIR /server
-
-COPY --chown=ue:ue . /server
-USER root
-CMD ./startup.sh
-```
-
-### startup.sh bash script:
-
-```bash
-chown -R ue.ue $PF_SERVER_LOG_DIRECTORY
```

Comment:
This is the bash script i meant

Reply:
Indeed. I double checked, and we did not deem this file necessary, so it was omitted from the expanded Linux instructions.

Reply:
@pgilmorepf we should include it to make writing logs successfully from within the GSDK.

Reply:
This script gives permission to the Unreal user to be able to write to /data/GameLogs